commit 45edd7e5c333ea5dd90e3a9814b1f30216dda667
Author: Luigi Toscano <luigi.toscano@tiscali.it>
Date:   Tue Mar 3 23:42:03 2015 +0100

    More up-to-date image for web pages (forgotten cmake fix)
    
    The new image is now installed; followup of review 122080.
    
    This change is not needed in Frameworks due to the different way
    CMakeLists.txt works there.

diff --git a/doc/common/CMakeLists.txt b/doc/common/CMakeLists.txt
index 4228625..0380fe9 100644
--- a/doc/common/CMakeLists.txt
+++ b/doc/common/CMakeLists.txt
@@ -29,5 +29,6 @@ install(FILES
     top-middle.png
     top-right.png
     top-left.png
+    part_of_the_kde_family_horizontal_190.png
     1.png 2.png 3.png 4.png 5.png 6.png 7.png 8.png 9.png 10.png
     DESTINATION ${HTML_INSTALL_DIR}/en/common)
